db/models.py

```diff
@@ -14,3 +14,41 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
+    genres = models.ManyToManyField(Genre, related_name="genres")
+
+    def __str__(self) -> str:
+        return self.title
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    def __str__(self) -> str:
+        return self.name
+
+    @property
+    def capacity(self) -> int:
+        return self.rows * self.seats_in_row
+
+
+class MovieSession(models.Model):
+    show_time = models.DateTimeField()
+    cinema_hall = models.ForeignKey(
+        CinemaHall,
+        related_name="hall",
+        on_delete=models.CASCADE)
+    movie = models.ForeignKey(
+        Movie,
+        related_name="movie",
```

Comment:
The `related_name` for the `movie` field in the `MovieSession` model is set to 'movie', which is not unique and can cause confusion. Consider using a more descriptive name like 'sessions' to indicate the reverse relationship from Movie to MovieSession.